src/index.ts

```diff
@@ -1,5 +1,4 @@
-import translations from './messageDescriptors';
+import translations from './messages';
 
 // eslint-disable-next-line import/no-default-export
-export default translations;
 export { translations };
```

Comment:
Having both default and named exports doesn't work for CommonJS-compatible output. In CommonJS the default export is the value assigned to `module.exports`, but transpiling TS `export default translations` produces `module.exports = { default: translations }`, which is not what one would expect.

One option to fix it would be to introduce two TS entry points: one for ESM (index.mts: `export default translations`) and another one for CJS (index.cts: `export = translations`). 

But the simplest solution is to just use a named export to make sure it's cross-compatible between CommonJS and ESM.